index.bs

```diff
@@ -1453,10 +1453,15 @@ URLs, is as follows:
 # Security Considerations # {#security-considerations}
 
 The Sanitizer API is intended to prevent DOM-based Cross-Site Scripting
-by traversing a supplied HTML content and removing elements and attributes
-according to a configuration. The specified API must not support
-the construction of a Sanitizer object that leaves script-capable markup in
-and doing so would be a bug in the threat model.
+by traversing supplied HTML content and removing elements and attributes
+according to a configuration. The Sanitizer API ships a strict baseline,
+such that scripting can never be allowed (cf. [[#never-allowed]]).
+In addition to this, a default safe list includes further restrictions that may help
+prevent a wide range of undesirable effects. These include, elements that can override
+site-specific settings (e.g., `<meta>`), embed third-party content or change the
+layout of the page (e.g., `<img>`, `<style>`).
+In addition, we disallow element and attribute configurations that lead to outgoing HTTP
+requests, and form controls. A detailed list is given in [[#default-disallowed-elements]].
```

Comment:
I think we should add a qualifier of "without a click" to the outgoing requests.

Also seems good to call out ARIA here.